Sources_v3/StreamChatUI/MessageActionsPopup/ChatMessagePopupViewController.swift

```diff
@@ -97,6 +129,20 @@ open class ChatMessagePopupViewController<ExtraData: ExtraDataTypes>: ViewContro
         messageContentView.reactionsBubble.isHidden = true
     }
 
+    override open func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        let contentOffset = CGPoint(x: 0, y: max(0, -messageViewFrame.minY + spacing + reactionsViewHeight))
+        scrollView.setContentOffset(contentOffset, animated: false)
+    }
+
+    override open func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        let contentRect = scrollContentView.convert(contentView.frame, to: scrollView)
+        scrollView.scrollRectToVisible(contentRect, animated: true)
+    }
```

Comment:
Can we make this in `viewWillAppear` such that the animation happens at the same time as the presentation animation? It will look much nicer to have it in a single step.

Reply:
I see the autolayout is not done when `viewWillAppear` is called. Let's leave as is then and polish later.